README.md

```diff
@@ -64,9 +64,9 @@ FLSlice greeting = FLValue_AsString( FLDict_Get(readProps, FLStr("greeting")) );
 CBLDocument_Release(readDoc);
 ```
 
-### Others
+### C++
 
-**NOTE**: The C++ API is not part of the official release.
+**NOTE**: The C++ API is a [volatile](https://docs.couchbase.com/couchbase-lite/3.0/android/compatibility.html#interface-stability) API which means that the API is not finalized, and may change in future releases.  
```

Comment:
Suggest pointing to the C docs link instead of Android docs https://docs.couchbase.com/couchbase-lite/3.0/c/compatibility.html#interface-stability

Reply:
Good catch!